.circleci/config.yml

```diff
@@ -116,6 +116,12 @@ workflows:
           filters:
             branches:
               only: master
+      - datadog-synthetics:
+          context:
+            - web-ui-datadog
+          requires:
+            - deploy
```

Comment:
I am not sure this is the right way to go. At this point, we are only running the tests after it deploys on `master`. If we broke something it's already too late.

I would really like it if we could test during the PR stage so we can have a signal that our proposed changes are about to break something. I was under the impression that we could tell the CLI to overwrite the test's URL: https://docs.datadoghq.com/synthetics/cicd_testing/?tab=npm#further-configuration

Anything we can do there?